docker/Dockerfile.buster

```
@@ -4,16 +4,24 @@ RUN useradd --user-group --create-home zulip
 USER zulip
 WORKDIR /home/zulip
 
-RUN set -ex; python3 -m venv zulip-terminal-venv \
-&& . zulip-terminal-venv/bin/activate \
-&& pip3 --disable-pip-version-check install zulip-term \
-&& rm -rf /home/zulip/.cache
+ARG SOURCE
```

Perhaps it is useful if this is the default?
```suggestion
ARG SOURCE=pypi
```

---

I did wonder about this, but thought it might be better to leave it needing to be explicit? I suppose we could also change `pypi` to `latest_release` or similar, in case users aren't familiar with what pypi is.

Re the `ZT_BRANCH` issue, we could have `GIT_URL`?
```
ARG GIT_URL=https://github.com/zulip/zulip-terminal.git@master
```
and later
```
    then pip3 --disable-pip-version-check install git+$GIT_URL ;

---

> I did wonder about this, but thought it might be better to leave it needing to be explicit?

I think the opposite, I think it would be a better "default experience" if
```
docker build -t zulip-terminal:latest -f Dockerfile.alpine .
```
worked. Messing with `--build-arg`s seems unnecessary if you just want to build the image and get started IMO.

---

@fredrikekre To clarify:
* I think it would be reasonable to take the approach you suggest - perhaps describing this in a "quick start" section with no build args needing to be specified, and then listing the git or pypi options?
* Was the :+1: to the `GIT_URL` approach?

@sumanthvrao What do you think?

---

> I think it would be reasonable to take the approach you suggest - perhaps describing this in a "quick start" section with no build args needing to be specified, and then listing the git or pypi options?

Okay, I misunderstood then. Sounds good.

> Was the +1 to the GIT_URL approach?

:+1: